packages/utils/src/lib/string.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Converts a kebab-case string to camelCase.
+ * @param string - The kebab-case string to convert.
+ * @returns The camelCase string.
+ */
+export function kebabCaseToCamelCase(string: string) {
+  return string
+    .split('-')
+    .map((segment, index) =>
+      index === 0
+        ? segment
+        : segment.charAt(0).toUpperCase() + segment.slice(1),
+    )
+    .join('');
+}
+
+export type CamelCaseToKebabCase<T extends string> =
+  T extends `${infer First}${infer Rest}`
+    ? Rest extends Uncapitalize<Rest>
+      ? `${Lowercase<First>}${CamelCaseToKebabCase<Rest>}`
+      : `${Lowercase<First>}-${CamelCaseToKebabCase<Rest>}`
+    : T;
```

Comment:
Our shared type helpers usually live in `types.ts`.

Reply:
done